### PR TITLE
feat: Adding in fake RSRP and Location service data

### PR DESCRIPTION
**Pull request by sherretten**
### Summary
When working with location service data on the FE we currently only have GPS/GNSS to test the location service map with. The goal of this is to add some points around Portland that send random AppId for location services [MCELL, SCELL, WIFI].

Additionally this add's RSRP support.

### Testing
1. Checkout this branch
2. Log into dev.nrfcloud.com
3. `yarn build`
4. `node dist/cli.js -k <API-key> -h https://api.dev.nrfcloud.com -a -d <DEVICE-ID> -s rsrp,location`
5. Go to the device view, should see points on the map with uncertainty circles

### Notes
* The BE isn't a fan of the appids MCELL and SCELL, so the device terminal will show some `50010` errors, not sure how to fix.

ticket: https://projecttools.nordicsemi.no/jira/browse/IRIS-4526

### CODE REVIEW

File: src/app/services/Location.ts
Comment (lincolnthomas) [inline src/app/services/Location.ts:6]: `CELL_POS` is listed in `src/app/appMessage.ts` enum `appId`, why not here?

File: src/sensors/FakeLocation.ts
Comment (lincolnthomas) [inline src/sensors/FakeLocation.ts]: Does this mean it will send a location every 3 seconds? 
That's probably too often, esp for the FE rendering the locations on the map. I've stopped using the `DEVICE` appId because it sends a message every 1 second, which is a relative flood that can cause problems after a while. The FE keeps shows incoming messages for a full minute after 3 minutes of the simulated device sending `DEVICE` messages, so things are backing up. Not sure where, but that's a subject for later. 

Every 10 seconds would be safer and fast enough, thoughts from anyone? Devs can edit their local code to make it more frequent when they want to.

So, unless anyone has qualms, could you also set the DEVICE default frequency to 5 seconds in this PR too?

**Review by lincolnthomas — CHANGES_REQUESTED** [review]
This has been waiting for a review for a while and it would be good to get these features in. @coleHafner / @savvyintegrations any review?

File: src/app/appMessage.ts
Comment (coleHafner) [inline src/app/appMessage.ts:14]: RSRP? It's a measurement of signal strength, not an AppID.

Comment (coleHafner) [inline src/app/appMessage.ts]: RSRP? It's a measurement of signal strength, not an AppID.

File: src/app/services/Rsrp.ts
Comment (coleHafner) [inline src/app/services/Rsrp.ts:19]: Is there something on the FE that looks for appID RSRP? It's not an officially recognized app on the backend....

**Review by coleHafner — COMMENTED** [review]
Code looks good. Had a few questions about the `RSRP` AppID.

File: src/app/appMessage.ts
Comment (lincolnthomas) [reply src/app/appMessage.ts]: It's not an App ID? Then [what is this](https://github.com/nRFCloud/application-protocols/blob/v1/schemas/deviceToCloud/rsrp/rsrp.json)?

Comment (coleHafner) [reply src/app/appMessage.ts]: I would have to defer to the author on that one...

File: src/app/services/Rsrp.ts
Comment (lincolnthomas) [reply src/app/services/Rsrp.ts:19]: It's totally independent of Location Services, if that's what you're thinking @coleHafner . Here's an example from a Thingy91:
```
{
"appId":"RSRP"
"messageType":"DATA"
"ts":1663343701739
"data":"-83"
}
```
And it has its own RSRP card on the device page, including History.
If the simulator sends something similar, then we're good. Another thing we can simulate besides LS stuff, thanks @sherretten !

Comment (sherretten) [reply src/app/services/Rsrp.ts:19]: Yeah that's what the simulator is sending. We use it on the FE for connection strength.

File: src/app/services/Location.ts
Comment (sherretten) [reply src/app/services/Location.ts:6]: I will remove it from `appMessage.ts`. 

The BE sends `MCELL` and `SCELL` with appId `CELL_POS` with live data but sends it as `MCELL` and `SCELL` with the location history table, so the FE supports both. I was thinking about a way to make this tool use the `CELL_POS` appId, but it adds more logic then I thought was necessary.

**Review by lincolnthomas — APPROVED** [review]
Changes LGTM

**Comment by lincolnthomas** [comment]
@sherretten Can you merge this now?

**Pull request merged**
